website/app/controllers/show.js

```diff
@@ -142,7 +143,8 @@ export default class ShowController extends Controller {
       tocs.push({
         index: 0,
         id: 'toc-all',
-        list: getAnchoredHeadings(container),
+        // we show only the headings level 2 and 3 in the sidecar
+        list: getAnchoredHeadings(container).filter((item) => item.depth <= 3),
```

Comment:
works for a hotfix; alternatively, we could update the `getAnchoredHeadings` to get a second argument (specifying the header limit or interval) but it would require a bit of refactoring within the function too

Reply:
Yeah, you're right. I like your idea. But I've looked at the code, and the `getAnchoredHeadings` does what it says on the tin, is pretty agnostic, so adding logic for filtering the returned array based on a second argument would not be too great. I'll live with the code (filter) duplication for now (hopefully the double comment will make it more visible that both need to be updated)